resources/js/cases/components/Tabs.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="hidden sm:tw-block">
+    <div class="tw-border-b tw-border-gray-200">
+      <nav
+        class="tw-mb-px tw-flex tw-space-x-8"
+        aria-label="Tabs"
+      >
+        <template v-for="(tab, index) in tabs">
+          <a
+            v-if="tab.show"
+            :key="index"
+            :href="tab.href"
+            :class="[tabSelected === tab.current ? 'tw-border-blue-500 tw-text-blue-500'
```

Comment:
:class="[tabSelected === tab.current ? 'tw-border-blue-500 tw-text-blue-500'
                       : 'tw-border-transparent tw-text-gray-500 hover:tw-border-gray-600 hover:tw-text-gray-600',
                     'tw-whitespace-nowrap tw-border-b-2 tw-px-1 tw-py-4 tw-text-sm tw-font-medium'
            ]"
            
            change to other way for example
 :class={
     'tw-border-blue-500 tw-text-blue-500': tabSelected === tab.current,
     'tw-border-transparent tw-text-gray-500 hover:tw-border-gray-600 hover:tw-text-gray-600': tabSelected != tab.current
     }
 
 class='tw-whitespace-nowrap tw-border-b-2 tw-px-1 tw-py-4 tw-text-sm tw-font-medium"
    
if the text color tw-text-gray-500 is used on different nodes, it must be used on a parent or container div.